pkg/controller/podautoscaler/horizontal.go

```diff
@@ -553,14 +553,14 @@ func (a *HorizontalController) reconcileAutoscaler(hpav1Shared *autoscalingv1.Ho
 	currentReplicas := scale.Status.Replicas
 	a.recordInitialRecommendation(currentReplicas, key)
 
-	var metricStatuses []autoscalingv2.MetricStatus
-	metricDesiredReplicas := int32(0)
-	metricName := ""
-	metricTimestamp := time.Time{}
```

Comment:
why remove "metricTimestamp" here, isn't it better to initialize it befere use.

Reply:
Thanks. Done.